.github/workflows/markdown-links.yaml

```diff
@@ -0,0 +1,23 @@
+name: Check Markdown links
+
+on:
+  pull_request:
+    branches:
+      - main
+  schedule:
+    - cron: "0 9 * * 1" # At 09:00 on Monday
+
+permissions:
+  contents: read
+
+jobs:
+  markdown-link-check:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@3df4ab11eba7bda6032a0b82a6bb43b11571feac # v3.5.2
+      - uses: gaurav-nelson/github-action-markdown-link-check@v1
```

Comment:
```suggestion
      - uses: gaurav-nelson/github-action-markdown-link-check@d53a906aa6b22b8979d33bc86170567e619495ec # v1.0.15
```

Reply:
Just a minor change - we try to always pin dependencies when we can so that we don't pull in un-audited third party code